ga4gh/cli.py

```diff
@@ -76,13 +76,16 @@ def server_main(parser=None):
 ##############################################################################
 
 
-class AbstractSearchRunner(object):
+class AbstractQueryRunner(object):
     """
-    Abstract base class for search runner classes
+    Abstract base class for runner classes
```

Comment:
In what sense do you mean "abstract" here? This isn't in the sense of https://www.python.org/dev/peps/pep-3119/, or I think in the C++/Java sense...